### ISSUE TITLE
Use GitHub features for Metadata IG

### ISSUE THREAD

**Issue by rettinghaus** [music-encoding/metadata-ig#16]
At the moment we document pretty much in Google Docs and use Slack for coordination. We should use more Github to keep track of issues, discussions and our meeting notes. That way we have everything in place and can more easily link to open GitHub issues. Also I think, this would be more open and inclusive than a somewhat hidden Docs file.

**Comment by riedde** [comment]
Yes, I absolutely agree! Since we have this repo I try to keep the [Meetings-Project](https://github.com/orgs/music-encoding/projects/5) and the Issues actual, but I'm not happy with our todays double structure.
Bringing our thoughts more to Github will make interaction with the rest of the MEI Community much more easier.

**Comment by Dr-MAH** [comment]
Inspired by Peter Stadlers minutes of the MerMEId IG I opened a document in the Git Wiki called „agenda & minutes“
(https://github.com/music-encoding/metadata-ig/wiki/agenda-&-minutes)
Would this be a better solution than using google docs? 

If yes we could pin it in the slack channel, if not I’ll remove it :-)

**Comment by gucl-mu** [comment]
In general, I also think it's better if everything is in one place. I like the use of the wiki as a place where we store our protocols, like in the IG MerMEId. But I would keep the googledocs for the draft versions of the guidelines. I know it's a bit easier to have it here, but these are drafts and I don't think they need to end up on github until we release them. So let's say we upload all the reviewed and proofread drafts and the rest stays in googledocs?

**Comment by irmlindcapelle** [comment]
I also like to give some "critical" thoughts in this discussion: I also like the open discussion with the MEI community but I think that there must also be a place for thoughts that are discussed in the group but not in the whole community. 
Therefore I like Clemens' proposal to have a place for internal discussions like drafts or thoughts on papers and meetings etc. and a place for accepted decisions, texts etc. I think it is one purpose of the IGs to collect special know-how and deeper discussion in a part of MEI, but another purpose to give room for discussion outside the whole community on these topics.

**Comment by ahankinson** [comment]
Speaking as someone “outside” the IG that has an interest in reviewing changes that are proposed, it is very useful to be able to see the discussion and the thought process through which a decision was arrived at, prior to making it to the broader community. Quite often the questions that come out of proposals are of the nature of “did you look at X” or “do you know that this is already covered by Y”. 

So even having access to drafts and other working documents can be valuable to those outside the IG. 

Changes to the core schema are always a discussion with the whole community, rather than simply implementation of a “decision” by the Interest group. We value the expertise and try to maintain a space where people of all backgrounds feel they can actively contribute, but we need to balance this with the fact that decisions can have a much broader impact than those within the IG itself. Having an open discussion space is highly valuable to those “passive” members who have an interest in these issues, but may not have the time or expertise to devote to IG work.

**Comment by riedde** [comment]
Thank you all for your comments. For me it seems that we are on the right way! At the moment we link our agenda and drafts to the [Metadata IG Meetings project](https://github.com/music-encoding/metadata-ig/issues/16#:~:text=to%20keep%20the-,Meetings%2DProject,-and%20the%20Issues), so everybody could access. Just to point out: The issues we open and discuss here are placed in the metadata-ig repo. This is some kind of internal but It can also accessed by the whole community if somebody is interested.
For example the fingerprint-issue: We opened an issue in this repo and discussed it, so the discussion was documented/reviewable. When opening an issue in the main repo (music-encoding) I was able to link that "ig-internal" issue. That was very nice, because I hadn't to say "the ig thinks that fingerprint should be deleted". I could just refer to that documented discussion that has helped to review it.

I think this repo is "private enough" for those "internal" drafts. And when something is grown up we can easily open this to the whole community.

I also like @Dr-MAH's initiative. It is hard to find something in our google docs if it is not clear when it came up. Github has a search function that should make it much more easier if we use the wiki.
